cloudnet-modules/cloudnet-database-mongodb/src/main/java/de/dytanic/cloudnet/ext/database/mongodb/CloudNetMongoDBDatabaseModule.java

```diff
@@ -0,0 +1,65 @@
+package de.dytanic.cloudnet.ext.database.mongodb;
+
+import com.google.gson.reflect.TypeToken;
+import de.dytanic.cloudnet.common.document.gson.JsonDocument;
+import de.dytanic.cloudnet.database.AbstractDatabaseProvider;
+import de.dytanic.cloudnet.driver.module.ModuleLifeCycle;
+import de.dytanic.cloudnet.driver.module.ModuleTask;
+import de.dytanic.cloudnet.ext.database.mongodb.command.SetupCommand;
+import de.dytanic.cloudnet.module.NodeCloudNetModule;
+
+import java.lang.reflect.Type;
+import java.util.List;
+import java.util.concurrent.Executors;
+import lombok.Getter;
+
+@Getter
+public class CloudNetMongoDBDatabaseModule extends NodeCloudNetModule {
+
+    public static final Type TYPE = new TypeToken<List<String>>() {
```

Comment:
Replace with a right typetoken. This makes on every usage a new object in the memory.
Check this -> https://www.javadoc.io/doc/com.google.code.gson/gson/latest/com.google.gson/com/google/gson/reflect/TypeToken.html
And use the `get` methodes of the typetoken.
Of that what you need is `getParameterized`